### PR TITLE
NW6| Fikret Ellek | React Module Project | Feature Footer - Week1

### PR DESCRIPTION
**Pull request by fikretellek**
## Learners, PR Template

Self checklist

- [x] I have committed my files one by one, on purpose, and for a reason
- [x] I have titled my PR with COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME | WEEK 
- [x] I have tested my changes
- [x] My changes follow the [style guide](https://syllabus.codeyourfuture.io/guides/code-style-guide/)
- [x] My changes meet the [requirements](./README.md) of this task

## Changelist

Briefly explain your PR.

## Questions

Ask any questions you have for your reviewer.

### CODE REVIEW

File: src/components/Footer/FooterList.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import FooterListItem from "./FooterListItem.jsx";
```
Comment (fhkahin) [inline src/components/Footer/FooterList.jsx:1]: well done great works

**Review by fhkahin — APPROVED** [review]
well done

File: src/components/Footer/Footer.test.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    const footerElement = screen.getByTestId("footer");
+    expect(footerElement).toBeInTheDocument();
+
+    // codes from line 14 to 17 has a bug
+
+    // const footerStyles = window.getComputedStyle(footerElement);
+    // console.log("here", footerStyles.position);
+    // expect(footerStyles.position).toBe("fixed");
+    // expect(footerStyles.bottom).toBe("0");
```
Comment (Ara225) [inline src/components/Footer/Footer.test.jsx:17]: I rember we discussed this bit of code in the class. I should have flagged that it's best to clean up commented out code when you can if it doesn't serve a purpose. Wouldn't take testing too seriously at this stage either they're just trying to get you into a habit, so there's no point spending ages on trying to write a perfect test, just write something :)

File: src/components/Footer/Footer.test.jsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  });
+  it("renders correctly with addresses displayed as list items", () => {
+    render(<Footer />);
+
+    const addresses = screen.getAllByTestId("address");
+    expect(addresses.length).toBeGreaterThan(0);
+    addresses.forEach((address) => {
+      expect(address.tagName).toBe("LI");
```
Comment (Ara225) [inline src/components/Footer/Footer.test.jsx:25]: I like this test though 👍

**Pull request merged**